examples/clipboard.py

```diff
@@ -0,0 +1,18 @@
+import pyglet
+
+"""Simple example demonstrating how to get the text in the clipboard as well as putting text in the clipboard."""
```

Comment:
I understand we are still at the proof of concept stage, but I think we could improve UX a lot with very little effort by:

1. Providing instructions at the top of the file
2. Separating copy and paste by assigning them to the standard platform hotkeys

This would make both testing and debugging easier by:
1. Separating copy and paste actions for increased control
2. Mapping clean to existing muscle memory

```suggestion
"""
How to get and set the text of the system clipboard.

This program does the following on launch:

1. Read the clipboard
2. Print the contents to the terminal

Afterward, you can use the normal copy and paste hotkeys for
your platform when the pyglet window has focus:

* Press CTRL-V to print text from the clipboard to the terminal
* Press CTRL-C to set the clipboard to 'Hello world!'
* If you are on Mac, use CMD instead of CTRL.

Output recording actions will be printed to the terminal.
"""
import pyglet
```